conformance/tests/specialtypes_promotions.py

```diff
@@ -10,7 +12,20 @@
 
 
 def func1(f: float):
-    f.numerator  # E
+    f.numerator  # E: attribute exists on int but not float
+
+    if isinstance(f, float):
+        assert_type(f, float)
```

Comment:
It's not actually clear to me what the behaviour of assert_type should be

Reply:
Oh right, because the second argument to `assert_type()` is itself a type expression. I'll adjust this test.